### PR TITLE
Test ac update

### PR DESCRIPTION
**Pull request by SoapGentoo**

### CODE REVIEW

**Comment by SoapGentoo** [comment]
@evpobr cleaning up before the ABI fix

**Comment by evpobr** [comment]
Why disable static build?

**Comment by SoapGentoo** [comment]
_by default_, you can't disable static libraries in libtool. The point is, don't produce static libraries unless the user asks for them

File: configure.ac [hunk 1]
```diff
@@ -54,7 +54,7 @@ dnl Catch ancient versions of pkg-config below 0.27
 AX_REQUIRE_DEFINED([PKG_INSTALLDIR])
 PKG_INSTALLDIR
 
-LT_INIT
+LT_INIT([disable-static])
```
Comment (evpobr) [inline configure.ac:57]: I guess it is better to add info about new behavior to CHANGELOG.md.

**Review by evpobr — CHANGES_REQUESTED** [review]

File: configure.ac (hunk 1 above)
Comment (SoapGentoo) [reply configure.ac:57]: better now?

**Review by evpobr — APPROVED** [review]

**Pull request closed without merging**